manifests/v/voidtools/Everything/Alpha/1.5.0.1344a/voidtools.Everything.Alpha.locale.en-US.yaml

```
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.2.3 $debug=QUSU.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: voidtools.Everything.Alpha
+PackageVersion: 1.5.0.1344a
+PackageLocale: en-US
+Publisher: voidtools
+PublisherUrl: https://www.voidtools.com
+PublisherSupportUrl: https://www.voidtools.com/support/everything/
+PrivacyUrl: https://www.voidtools.com/privacy
+Author: voidtools
+PackageName: Everything (Alpha)
+PackageUrl: https://www.voidtools.com/forum/viewtopic.php?f=12&t=9787
+License: MIT License
+LicenseUrl: https://www.voidtools.com/License.txt
+Copyright: Copyright (C) 2023 voidtools

```

```suggestion
Copyright: Copyright (C) 2022 voidtools
```
Copyright year hasn't been updated at https://www.voidtools.com/License.txt, so I believe we should use whatever is in the CopyrightUrl